### ISSUE TITLE
Can't install add-on in Node 10 env

### ISSUE THREAD

**Issue by lougreenwood** [isleofcode/ember-cordova-installer#3]
Crosslink from https://github.com/isleofcode/ember-cordova-splash/issues/9

When attempting to install `ember-cordova-splash` in Node 10 env, Yarn throws this error:

```
error ember-cordova-installer@0.0.4: The engine "node" is incompatible with this module. Expected version "^4.5 || 6.* || 7.* || 8.* || 9.*".
error Found incompatible module
```

**Comment by drewcovi** [comment]
@lougreenwood did you find a workaround? @alexblom any thoughts on how we can get this resolved? having similar issues here.

**Comment by lougreenwood** [comment]
@drewcovi Yep, I use this branch/PR https://github.com/isleofcode/ember-cordova-splash/pull/14

in package.json:

```
    "ember-cordova-splash": "lougreenwood/ember-cordova-splash#bump-ember-cordova-installer",
```

**Comment by drewcovi** [comment]
thanks for making the PR. seems we're alone in the world of ember-cordova ... having splash control seems pretty central to building an app.
